File: tests/Generator/resource/ruby/complex/argument.rb

# Describes arguments of the operation
class Argument
  attr_accessor :content_type, :in, :name, :schema

  def initialize(content_type, in, name, schema)
    @content_type = content_type
    @in = in
    @name = name
    @schema = schema
  end
end

File: tests/Generator/resource/csharp/complex/Argument.cs
using System.Text.Json.Serialization;

namespace TypeAPI.Model;

/// <summary>
/// Describes arguments of the operation
/// </summary>
public class Argument
{
    [JsonPropertyName("contentType")]
    public string? ContentType { get; set; }

    [JsonPropertyName("in")]
    public string? In { get; set; }

    [JsonPropertyName("name")]
    public string? Name { get; set; }

    [JsonPropertyName("schema")]
    public PropertyType? Schema { get; set; }

}

